docs/content/concepts/assets/asset-auto-execution.mdx

```diff
@@ -30,13 +30,11 @@ from dagster import AutoMaterializePolicy, asset
 
 
 @asset
-def asset1():
-    ...
+def asset1(): ...
```

Comment:
hmm, looking at https://docs.dagster.io/concepts/assets/asset-auto-execution#auto-materializing-assets- I feel like these collapsing to the same line feels worse as a place holder for "real body of your fn" 

should we update these docs examples to use a comment `# compute asset 1` or `pass` ? or do others think this is fine and we should just proceed? This is definitely a style not substance thing.

Reply:
I definitely see where you're coming from here. I investigated a few options but I think the best is to disable `preview` mode in `docs_snippets`. This works because:

1. We already use a separate black config for docs_snippets, so the config change is trivial
2. This has almost no changes aside from disabling this rule
3. Ruff is developing a formatter that we will probably switch to in a few months, and ruff will likely offer much greater configurability than black for stuff like this. So when/if this feature makes it into the core style, by that time we'll probably be able to selectively disable it for docs_snippets.

Reply:
I just updated the PR with ^^ change in a separate commit.

Reply:
i still see the changes in mdx - need to re-run the docs build thing too? 

Other than that this approach sounds good to me

Reply:
>i still see the changes in mdx - need to re-run the docs build thing too?

oops, updated

Reply:
speak of the devil: https://github.com/astral-sh/ruff/issues/1904#issuecomment-1716130700